src/commands/install-nancy.yml

```diff
@@ -0,0 +1,25 @@
+parameters:
+  nancy-version:
+    type: string
+    default: v0.0.41
+    description: >
+      The version of Nancy to run. See: https://github.com/sonatype-nexus-community/nancy/releases for available
+      versions.
+  nancy-platform:
+    type: string
+    default: linux.amd64
+    description: >
+      The underlying OS on which Nancy will run. Valid values are: linux.amd64, darwin.amd64, windows.amd64.
+      See: https://github.com/sonatype-nexus-community/nancy/releases for all available platforms.
+      Please let us know if you find a need for another platform, or even better, submit a PR!
+steps:
+  - run:
+      name: Install Nancy
+      command: |
+        cd /tmp && mkdir tools && cd -
+        sourceUrl="https://github.com/sonatype-nexus-community/nancy/releases/download/<< parameters.nancy-version >>/nancy-<< parameters.nancy-platform >>-<< parameters.nancy-version >>"
+        if [[ $sourceUrl == *"windows"* ]]; then
+          sourceUrl="${sourceUrl}.exe"
+        fi
+        curl --fail -s -L "$sourceUrl" -o "/tmp/tools/nancy"
```

Comment:
Will this work on Windowz?

Reply:
Hm. Maybe not, but at least the IT passes by being able to download the executable. ;)
Would have look into a Winz CI build to validate it. I'll create a new issue to track it.